### ISSUE TITLE
code runs very slow on nvidia jetson tk1

### ISSUE THREAD

**Issue by nehalb123** [AlexeyAB/darknet#485]
I ran the code on my laptop with only `OPENCV=1` in Makefile and other parameters 0.
I get an FPS around 0.6 on CPU.

I  ran the code with `GPU =1 and OPENCV=1` on Jetson TK1 but still I get FPS only of 0.1. 
Normal version of darknet gives FPS around 3.6 on Jetson tk1.

Why is this code slower than the vanilla version of darknet?

**Comment by sivagnanamn** [comment]
- Are you using `CUDNN=1` ?
- Which network cfg are you using?
- What is the input image resolution in the cfg?

**Comment by AlexeyAB** [comment]
> I ran the code with GPU =1 and OPENCV=1 on Jetson TK1 but still I get FPS only of 0.1.

After the settings `GPU =11 and 1OPENCV=1` in the Makefile are set, then do:
```
make clean
make -j8
```

And what command do you use for detection?

**Comment by nehalb123** [comment]
@AlexeyAB  command used: `./darknet detector demo cfg/coco.data cfg/yolo.cfg yolo.weights -i 0 -http_port 8090 -out_filename res.avi`

**Comment by AlexeyAB** [comment]
@nehalb123 Use this command:
`./darknet detector demo cfg/coco.data cfg/yolo.cfg yolo.weights -i 0`
How many FPS can you get?

**Comment by nehalb123** [comment]
@AlexeyAB ` FPS - 0.0` and in the above comment u mean `GPU=1` right?

**Comment by AlexeyAB** [comment]
> @AlexeyAB FPS - 0.0 and in the above comment u mean GPU=1 right?

Yes, GPU=1. And do you get 0 FPS without flags `-http_port 8090 -out_filename res.avi`?

**Comment by nehalb123** [comment]
`FPS -0.0`

**Comment by nehalb123** [comment]
Yes but in original darknet (https://pjreddie.com/darknet/yolo/) i get `FPS:3.6`

**Comment by MizbaMohammed** [comment]
Hi @AlexeyAB is there a way  to increase YOLO FPS on Tx2? When I ran YOLO v2 on my laptop I was able to achieve about 25 FPS but when I am running it on my Tx2 I can only achieve 6-7 FPS. Do you know why there is so much difference?

**Comment by AlexeyAB** [comment]
@MizbaMohammed You can get about 15 fps using tiny-yolo 416x416 on TX2 using this repo: https://github.com/AlexeyAB/darknet/issues/529#issuecomment-377235623

**Comment by MizbaMohammed** [comment]
@AlexeyAB do you think training the model to detect fewer objects might increase the FPS on TX2?

**Comment by AlexeyAB** [comment]
@MizbaMohammed FPS does not depend on the number of classes.
